mixer/adapter/solarwinds/README.md

```diff
@@ -0,0 +1,16 @@
+<p align="center"><strong>NOTE</strong>: this is considered <strong>EXPERIMENTAL</strong> and is not yet recommended for production systems.</p>
+
+# Istio Mixer Adapter
+An Istio Mixer adapter for integration with AppOptics and Papertrail. As a SolarWinds Innovation Project, this adapter is supported in a best-effort fashion.
+
+## Purpose
+This adapter allows you to bring your services’ logs and metrics collected through Istio Mixer into Papertrail and AppOptics, respectively, to enable all of the advanced search and organizational capabilities, giving users a broader look at their app’s performance. As apps become more distributed with microservices, it’s important to continue finding ways to help provide tools to show relevant metrics on both overall and isolated performance.
+
+### AppOptics integration screenshot
+![AppOptics integration screenshot](docs/istio-appoptics-2.png)
```

Comment:
There are 5 images in this PR, however README is only using two. Do we need the other extra pngs ? can we delete them ?

Reply:
@leecalcote fixed this for us.